public/app/core/components/Select/ServiceAccountPicker.tsx

```diff
@@ -0,0 +1,79 @@
+import { debounce, isNil } from 'lodash';
+import React, { Component } from 'react';
+
+import { SelectableValue } from '@grafana/data';
+import { getBackendSrv } from '@grafana/runtime';
+import { AsyncSelect } from '@grafana/ui';
+import { ServiceAccount } from 'app/types';
+
+export interface Props {
+  onSelected: (serviceAccount: SelectableValue<ServiceAccount>) => void;
+  className?: string;
+  inputId?: string;
+}
+
+export interface State {
+  isLoading: boolean;
+}
+
+export class ServiceAccountPicker extends Component<Props, State> {
```

Comment:
This is deprecated way to create new components, it's better to use functional components, for example, it might be like OrgPicker in the same folder.

Reply:
Thanks, I'll take a look at that 👍

Reply:
I've updated the code, but I have little confidence in my frontend changes 🙈 I'd appreciate if you could take another look at them.